apps/web/src/components/templates/in-app-editor/ContentContainer.tsx

```diff
@@ -17,30 +18,11 @@ export function ContentContainer({
 }) {
   const theme = useMantineTheme();
   const ref = useRef<HTMLDivElement>(null);
-  const [visiblePlaceholder, setVisiblePlaceholder] = useState(!!value);
+  const showPlaceHolder = value.length === 0;
   const [content, setContent] = useState<string | IEmailBlock[]>(value);
 
-  const handleChange = (e: React.ChangeEvent<HTMLDivElement>) => {
-    const textContent = e.target.textContent;
-    if (ref?.current) {
-      ref.current.textContent = textContent;
-    }
```

Comment:
this line was causing the issue, setting the `textContent` removes the `the node's children` and replaces it with a new `text node`